Bottom: 56faf3861a89e361848f575a9180980e33175d58
Top:    a1a431acdfc15924ce55dd684833711b254d66e1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-29 13:50:16 +0000

Refresh of debug-changes

---

diff --git a/src/hid/common/face3d_gl.c b/src/hid/common/face3d_gl.c
index d372ac2..88833d6 100644
--- a/src/hid/common/face3d_gl.c
+++ b/src/hid/common/face3d_gl.c
@@ -134,9 +134,6 @@ emit_tristrip (face3d *face)
   glTexCoord2f (0.0f, 0.0f);
 
   glDrawArrays (GL_TRIANGLE_STRIP, 0, face->tristrip_num_vertices);
-
-//  glColor3f (1.0f, 1.0f, 1.0f);
-//  glDrawArrays (GL_LINE_STRIP, 0, face->tristrip_num_vertices);
 #if 0
   glPushAttrib (GL_CURRENT_BIT);
   glColor4f (1., 1., 1., 1.);
